cms/templates/home_page.html

```diff
@@ -20,6 +20,7 @@
 
 {% block content %}
   {% include "partials/hero.html" with action_title="Watch Now" action_url="#" %}
+  {% cache 86400 self.id %}
```

Comment:
I think 24 hours is a bit too aggressive, how about we try 5 minutes? At least then we know it won't be every request but if we need to clear the cache we only have to wait a short time